core/src/main/scala/cats/arrow/Compose.scala

```diff
@@ -41,9 +41,12 @@ import scala.annotation.implicitNotFound
 }
 
 object Compose {
-  implicit def catsInstancesForFunction1: ArrowChoice[Function1] with CommutativeArrow[Function1] =
+  implicit val catsInstancesForFunction1: ArrowChoice[Function1] with CommutativeArrow[Function1] =
```

Comment:
It's not really a big deal, but there is a reason for these to be methods instead of `val`s: they're already instantiated as `val`s in the `instances` objects, so using `def` here won't result in new instantiations, but using a `val` does result in a little extra overhead and indirection, since it generates both an accessor method and a field.

In general this is unlikely to matter much, but because there are a lot of these, we've had a policy of `def` instead of `val`.

Reply:
I see, that's a nice way of saving up on a reference; apologies for oversight